src/test/java/org/folio/linked/data/mapper/kafka/search/BibliographicSearchMessageMapperTest.java

```diff
@@ -91,61 +93,58 @@ public void setupMocks() {
     ).forEach(t ->
       lenient().when(singleResourceMapper.getMapperUnit(eq(t), any(), any(), any())).thenReturn(of(genericMapper()))
     );
-    kafkaMessageMapper = new BibliographicSearchMessageMapper(
-      new BibframeInstancesInnerIdentifiersInnerMapperAbstract(), singleResourceMapper);
   }
 
   @Test
-  void toIndex_shouldReturnCorrectlyMappedIndex_fromWork() {
+  void toIndex_shouldReturnCorrectlyMappedIndex_fromResourceWithIdOnly() {
     // given
-    var work = getSampleWork(null);
-    var wrongContributor = getContributor(ANNOTATION);
-    var emptyContributor = new Resource();
-    work.addOutgoingEdge(new ResourceEdge(work, wrongContributor, CONTRIBUTOR));
-    work.addOutgoingEdge(new ResourceEdge(work, emptyContributor, CONTRIBUTOR));
-    final var instance1 = getInstance(1L, work);
-    final var instance2 = getInstance(2L, work);
+    var resource = new Resource().setId(randomLong());
 
     // when
-    var resultOpt = kafkaMessageMapper.toIndex(work, CREATE);
+    var result = kafkaMessageMapper.toIndex(resource);
 
     // then
-    assertThat(resultOpt).isPresent();
-    var result = resultOpt.get();
-    validateWork(result, work, wrongContributor, 2);
-    validateInstance(result.getInstances().get(0), instance1);
-    validateInstance(result.getInstances().get(1), instance2);
+    assertThat(result)
+      .hasAllNullFieldsOrPropertiesExcept("id", "resourceName", "_new")
+      .hasFieldOrPropertyWithValue("id", String.valueOf(resource.getId()))
+      .hasFieldOrPropertyWithValue("resourceName", "linked-data-work")
```

Comment:
minor: Constants.SEARCH_RESOURCE_NAME

Reply:
It's better to nut use production code in a unit test